README.rst

```diff
@@ -1,3 +1,5 @@
+.. role:: bash(code)
+   :language: bash
```

Comment:
This isn't really being used for bash. Is there a ``:code:`` role? If not, note that ``things`` is literal (i.e., default formatted monospace) in rst.

Reply:
Oh I see, so will it format with styling using only \`words\` ? When I tested earlier it didn't work.

Reply:
It needs two backticks on either side, and then it should. So, it'd still be a change, but wouldn't need the new role.